### ISSUE TITLE
Import results in error for related field

### ISSUE THREAD

**Issue by adamcooley** [django-import-export/django-import-export#459]
I've got a related field that is required for an import, and on 0.4.5 I'm getting an error saying that the "RelatedObjectDoesNotExist" any ideas?

Exception:

```

Line number: 1 - AssetInventory has no sku.
, rsw001, 1000000.0, SN10000, EX4300-32F, , , , , , , , , , , 42503.0, , , RECEIVED, , , , , ,

Traceback (most recent call last):
File "/Library/Python/2.7/site-packages/import_export/resources.py", line 414, in import_row
row_result.object_repr = force_text(instance)
File "/Library/Python/2.7/site-packages/django/utils/encoding.py", line 78, in force_text
s = six.text_type(s)
File "/Users/acooley/Dropbox (Personal)/Software/Projects/nemesis/asset/models.py", line 23, in __unicode__
return str(self.sku.Type) + " Tag: " + str(self.assetTag)
File "/Library/Python/2.7/site-packages/django/db/models/fields/related_descriptors.py", line 179, in __get__
"%s has no %s." % (self.field.model.__name__, self.field.name)
RelatedObjectDoesNotExist: AssetInventory has no sku.
```

**Comment by ezubillaga** [comment]
Check issue #439. The proposal solution there did the trick for me

**Comment by adamcooley** [comment]
yeah modifying the __unicode__ for this model to just return a string for a non related field works fine. Im not exactly sure why this is happening, would a feature request be ideal here? Or is this expected behaviour?

**Comment by JarnoRFB** [comment]
As it seems, the models `__str__` or `__unicode__` method is used as an attribute of the `row_result` object, but I think this only internal and never presented to the user, so maybe a pull request would be good idea. Instead of using `try ... except` in the models method, it could be used directly in the script to use a placeholder if `row_result.object_repr = force_text(instance)` throws an error. But maybe `row_result.object_repr` has another important function that I'm missing here.

**Comment by jschneier** [comment]
This should be fixed in master.

**Comment by adamcooley** [comment]
Cool thanks!

**Comment by int-ua** [comment]
Fixed in `0.6.0` with #505
